interface/super/edit_globals.php

```diff
@@ -50,7 +54,7 @@ function checkCreateCDB(){
       $GLOBALS[$globalsrow['gl_name']] = $globalsrow['gl_value'];
     }
     $directory_created = false;
-  if( !empty($GLOBALS['document_storage_method']) ) {
+  if($GLOBALS['document_storage_method'] != 0){
```

Comment:
Any takers?  Please someone do a review?  It's a single-file review, not hard...